.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,92 @@
+                        line.strip(),
+                    ):
+                        print(
+                            f"❌ Error: '@ts-ignore' found in {file} "
+                            f"at line {line_num}"
+                        )
+                        logging.debug(
+                            f"Found @ts-ignore in line: {line.strip()}"
+                        )
+                        ts_ignore_found = True
+        except FileNotFoundError:
+            logging.warning(f"File not found: {file}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)